### PR TITLE
Add ThreePanesView extension

### PR DESCRIPTION
**Pull request by nicofrand**
Hi,

I feel like the extension is stable enough to list it here now :).

Please let me know if not.

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -120,6 +120,14 @@
       "author": "Lapineige",
       "url": "https://framagit.org/Lapineige/FreshRSS_Extension-ReadingTime/",
       "type": "gh-subdirectory"
+    },
+    {
+      "name": "ThreePanesView",
+      "version": 1.2,
+      "description": "Adds a third vertical pane along the articles list, to display the articles content.",
```
Comment (Frenzie) [inline extensions.json]:
```suggestion
      "description": "Adds a third vertical pane along the article list, to display the article's content.",
```

**Comment by nicofrand** [comment]
Well, I think both are correct. With the plural form I meant that it displays the content of all (any of) the articles.. But ofc it only displays one at a time..


If needed I'll amend it.


Le 19 févr. 2019 20:43, Frans de Jonge <notifications@github.com> a écrit :

@Frenzie commented on this pull request.

_____________________________________________

In extensions.json:

**Comment by nicofrand** [comment]
Ah, I did not see the "article list", I'll change it.


Le 19 févr. 2019 20:43, Frans de Jonge <notifications@github.com> a écrit :

@Frenzie commented on this pull request.

_____________________________________________

In extensions.json:

**Comment by Frenzie** [comment]
You can also just click a button here on GH to automatically incorporate the change.

**Comment by nicofrand** [comment]
The button did not work on Firefox for Android :/.

**Comment by Frenzie** [comment]
Thanks!

I've experienced something similar in the past. They're probably just nudging you to use a proper computer. ;-D

**Pull request merged**